tests/chai.js

```diff
@@ -1,14 +0,0 @@
-'use strict';
```

Comment:
Moving this to `tests/runner.js` enforces consistent importing of `chai` and `chai-files`.